registry/sql-registry/registry/db_registry.py

```diff
@@ -405,16 +405,18 @@ def _fill_entity(self, e: Entity) -> Entity:
         return e
 
     def _get_edges(self, ids: list[UUID], types: list[RelationshipType] = []) -> list[Edge]:
-        if not ids or not types:
+        if not ids:
             return []
         sql = fr"""select edge_id, from_id, to_id, conn_type from edges
         where from_id in %(ids)s
         and to_id in %(ids)s"""
+        types = [ str(t) for t in types]
```

Comment:
Duplicated with line 422.
Also should use `name` property for `Enum` objects.

Reply:
Added for debugging. Will remove it.